examples/pytorch/multigpu/multi_gpu_graph_prediction.py

```diff
@@ -0,0 +1,141 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.distributed as dist
+import torch.optim as optim
+import dgl
+import dgl.nn as dglnn
+from dgl.data import AsGraphPredDataset
+from dgl.dataloading import GraphDataLoader as DataLoader
```

Comment:
This is a bit strange. Just use `GraphDataLoader` is fine. No need to create alias.

Reply:
fixed